### ISSUE TITLE
Question - Bail mode

### ISSUE THREAD

**Issue by kimchi-lover** [vinejs/vine#87]
Hello. It looks like bail(false) doesn't work properly (or maybe I didn't get a point what it actually does).
For example, let's look at this code:

```javascript
import vine from '@vinejs/vine'

const schema = vine.object({
  email: vine.string().email().minLength(5).bail(false)
})

const data = {
  email: 'virk',
}

const validator = vine.compile(schema)
const output = await validator.validate(data)

console.log(output)
```

Here I expect to get both errors(incorrect email and about length), but I have got only first one(about email).
Is this a bug or I just don't understand how it's working?

**Comment by thetutlage** [comment]
Indeed. That looks like an issue with the rules implementation. It should not check for the field validity internally and rely on the compiler + bail mode to skip invalid fields.

I will handle it 👍

**Comment by enso-san** [comment]
@thetutlage Any update when this will be fixed?
